### ISSUE TITLE
Cannot verify contract - Etherscan

### ISSUE THREAD

**Issue by nufailTanjiro** [ethereum/remix-project#5130]
**Describe the bug**
When I verify my contract using the ethersan contract verification plugin. Even though my etherscan API key is correct I get Invalid Key

**To Reproduce**
1. Go to etherscan contract verification
2. Fill up the details
3. Click Verifiy 
4. I get Invalid API Key

Checked my API Key as well. It's correct

**Expected behavior**
Should verify the contract

**Screenshots**
![image](https://github.com/user-attachments/assets/13194a25-5a9d-49dd-845d-6fae16699689)

**Desktop (please complete the following information):**
 - OS:  Windows
 - Browser: chrome (Brave)

**Comment by Aniket-Engg** [comment]
Are you sure you have added correct API key to Remix Etherscan plugin?

**Comment by NufailIsmath** [comment]
Yes, that's my API key, in fact I created new one tested. Just in case

**Comment by nufailTanjiro** [comment]
Yes i double checked my API key

**Comment by crazywidmer** [comment]
I'm having the same issue

**Comment by Aniket-Engg** [comment]
@nufailTanjiro We have released a new plugin `Contract Verification`. Can you try verifying using that?

**Comment by nufailTanjiro** [comment]
Sure, I'll try with that. By the way, it's the same process as the previous plugin Yeah, no additional steps to verify the contract?

**Comment by cheryllynn7327** [comment]
Ok cause idk what I’m doing yet
